src/components/Documents/MyDocuments.tsx

```diff
@@ -357,6 +357,7 @@ class MyDocuments extends Component<Props, State> {
                           data={documentData}
                           columns={this.tableCols}
                           emptyInfo={{ description: 'No documents found' }}
+                          defaultSorted={[{ dataField: 'uploadDate', order: 'asc' }]}
```

Comment:
It seems like it should be descending, but that gives the opposite of what we want. I think this is a bug in the react-bootstrap-table